docs/index.rst

```diff
@@ -45,6 +45,16 @@ See :doc:`Installation <installation>` for more information.
 .. _cryptography-layout:
 
 
+Thread Safety
```

Comment:
You asked where the right place for this would be: My suggestion is adding a new term in the glossary, which we can then reference in various places. (e.g., you asked about fernet, so we can add a reference there).

Reply:
Just moved the section to the glossary. Is this what you had in mind?